tests/ops/test_qubit_ops.py

```diff
@@ -987,12 +989,30 @@ def test_single_excitation_matrix(self, phi):
 
     @pytest.mark.parametrize("phi", [-0.1, 0.2, np.pi / 4])
     def test_single_excitation_decomp(self, phi):
-        """Tests that the SingleExcitation operation calculates the correct decomposition"""
-        op = qml.SingleExcitation(phi, wires=[0, 1])
-        decomp = op.decomposition(phi, wires=[0, 1])
-        mats = [m.matrix for m in decomp]
-        decomposed_matrix = mats[0] @ mats[1]
+        """Tests that the SingleExcitation operation calculates the correct decomposition.
+
+        Need to consider the matrix of CRY separately, as the control is wire 1
```

Comment:
Why is this change to the single excitation gate being done in this PR? Just sneaking it in?

Reply:
It's because we are updating the gradient recipes for both gates to the four-term shift rules; so the `SingleExcitation` decomposition is adjusted accordingly.